api/envoy/admin/v3/config_dump.proto

```diff
@@ -17,6 +17,28 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 
 // [#protodoc-title: ConfigDump]
 
+// Config status from the view of a xDS client, which tells the synchronization
+// status between the xDS client and the xDS server.
+enum ConfigErrorStatus {
```

Comment:
Suggest replacing the term "config" with "resource" throughout (enum name, field names, comments, etc), since this is really about per-resource status.

Reply:
The norm of term within this file is using "config" to describe a "resource". Personally, I found the term "resource" could map to either an instance of a xDS config or a type of xDS config. Within this file, I would vote for keeping the term "config".

Reply:
My bad. It seems this file is already mixing "config" and "resource". I will update the term.